whatrecord/common.py

```diff
@@ -596,7 +596,7 @@ def update_unknowns(
 
 def get_link_information(link_str: str) -> Tuple[str, List[str]]:
     """Get link information from a DBF_{IN,OUT,FWD}LINK value."""
-    if isinstance(link_str, dict):
+    if not isinstance(link_str, str):
         # Oh, PVA...
         raise ValueError("PVA links are TODO, sorry")
```

Comment:
Is this error text out of date? We raise on non-str types now instead of on structured data

Reply:
It's a fair point: it turned into a catch-all for JSON links or structured PVA data

I think I'd like to adjust this and then merge